### PR TITLE
Debug Options for Email Popup Issue

### PR DESCRIPTION
**Pull request by okariJC**
Some debug options for the EmailPopup Issue... Cleaned up so they can go into a release if need be.

### CODE REVIEW

File: Assets/Local/Scripts/SROptions.cs [hunk 1]
```diff
@@ -68,6 +70,20 @@ public void InfoDumpToClipboard()
         Application.OpenURL(mailURL);
     }
 
+    [Category("Popup Rebuilding Controls")]
```
Comment (jeremyfJC) [inline Assets/Local/Scripts/SROptions.cs]: are these specific to your JCID?

File: Assets/Plugins/Jamcity/JamCityID/Scripts/PopupMessage.cs [hunk 2]
```diff
@@ -54,7 +70,34 @@ public void Show(PopupMessageContext context)
 
             labelBodyText.text = translationService.Translate(context.BodyKey, context.BodyArguments);
             gameObject.SetActive(true);
-            LayoutRebuilder.ForceRebuildLayoutImmediate(GetComponent<RectTransform>());
+
+            if (forceRebuildLate.Value)
```
Comment (jeremyfJC) [inline Assets/Plugins/Jamcity/JamCityID/Scripts/PopupMessage.cs]: can we just use this value, instead of the `forceRebuild`?

Comment (okariJC) [reply Assets/Plugins/Jamcity/JamCityID/Scripts/PopupMessage.cs]: nope, this is a configural options from an SO

forceRebuild is so lateupdate knows to force rebuild the once.

File: Assets/Local/Scripts/SROptions.cs (hunk 1 above)
Comment (okariJC) [reply Assets/Local/Scripts/SROptions.cs]: yes not for clients

**Pull request merged**